test/local/metrics/amplitude.js

```diff
@@ -248,6 +248,7 @@ describe('metrics/amplitude', () => {
         assert.equal(log.amplitudeEvent.callCount, 1)
         const args = log.amplitudeEvent.args[0]
         assert.equal(args[0].event_type, 'fxa_login - blocked')
+        assert.equal(args[0].event_properties.service, 'sync')
```

Comment:
Wasn't apparently to me if there was a test when service falls back to using service id instead of name.

Reply:
Yep there is, it just doesn't show up in the diff because that code didn't need to change.